### ISSUE TITLE
Unable to use Azure Storage Explorer on my Mac

### ISSUE THREAD

**Issue by saikongara** [microsoft/AzureStorageExplorer#3754]
Here is the error I am getting while opening Azure Storage explorer

Unable to retrieve child resources.

Details:
["Error:Unable to retrieve authentication token."]

**Comment by MRayermannMSFT** [comment]
Can you go to Settings -> Services -> Disks, uncheck the checkbox and then restart? That should unblock you (assuming you don't need to work with disks).

Also, please check the account panel for any accounts which are asking you to re-enter your credentials, and if there are any, then do so.

**Comment by edtiong** [comment]
this worked for me

**Comment by MRayermannMSFT** [comment]
saikongara has another issue open so I'm closing this one

**Comment by saikongara** [comment]
This isn't helpful. I couldn't open any window after installing the application.
